emsdk.py

```diff
@@ -29,13 +28,9 @@
     import _winreg as winreg
   import ctypes.wintypes
 
-if sys.version_info >= (3,):
-  from urllib.parse import urljoin
-  from urllib.request import urlopen
-  import functools
-else:
-  from urlparse import urljoin
-  from urllib2 import urlopen
```

Comment:
This one script in particular we try to keep running on python2.  The reason is that this script actually downloads the version of python3 that we use in emsdk.   So its good to be able to bootstrap the system on systems that have python2 install by default (like older versions of macOS apparently).